deploy/role.yaml

```diff
@@ -47,8 +47,10 @@ rules:
   verbs:
   - '*'
 - apiGroups:
-  - ""
+  - "*"
   resources:
   - "*"
   verbs:
     - "get"
+    - "list"
+    - "watch"
```

Comment:
@otaviof , Funny, I needed to add the `watch` here to be run this PR as a `Deployment` in-cluster , and not `make local`